### ISSUE TITLE
Use of `"@vocab": "_:"` in ActivityStreams 2.0 (at least)

### ISSUE THREAD

**Issue by BigBlueHat** [w3c/json-ld-syntax#183]
I've been digging through as many `@context` files as I can find "in the wild"--especially ones in wide/popular/deployed usage--in hopes of better informing our changes and "upgrade" planning.

While digging, I uncovered this use of `@vocab` at the top of the [ActivityStreams](https://www.w3.org/ns/activitystreams.jsonld) context file:
```json
{
  "@context": {
    "@vocab": "_:",
    "...etc...": "...etc..."
  }
}
```

This ultimately means that anything not defined in [ActivityStreams 2.0's Vocabulary](https://www.w3.org/TR/activitystreams-vocabulary/) or not covered by an additional "extension" context definition will be mapped into a blank node.

There's some well informed discussion of the risks, rewards, and reasoning behind their choice:
https://github.com/w3c/activitystreams-testing/issues/4
(follow links there to past issues/resolutions).

Most of the discussions center around extensibility and some of those issues relate to the current work going into `@protected` (and friends).

**Comment by rubensworks** [comment]
Semi-related to this, I have a use case where I'm overriding `@vocab` to throw an explicit error on terms that could not be expanded when transforming to RDF (to avoid silently ignoring them).
In [this context](https://github.com/LinkedSoftwareDependencies/Components.js/blob/master/components/context.jsonld#L3) `@vocab` is set to `"unknown://"`.
After converting JSON-LD documents to RDF with this context, I iterate over all triples to detect terms starting with `unknown://`, and throw an explicit error for them.

**Comment by iherman** [comment]
This issue was discussed in [a meeting](https://www.w3.org/2018/json-ld-wg/Meetings/Minutes/2019/2019-06-28-json-ld#section4-2).

- `RESOLVED:  closed #183 as wontfix`
<details><summary><i class="differentiate">View the transcript</i></summary>
4.2. Use of "<code>@vocab":</code> "_:" in ActivityStreams 2.0 (at least) #183<br/>
Benjamin Young: <a href="https://github.com/w3c/json-ld-syntax/issues/183">https://github.com/w3c/json-ld-syntax/issues/183</a><br/>
Gregg Kellogg: We haven’t removed support for blank nodes, just marked it as archaic and possibly source of warning<br/>
… we haven’t broken activity streams, but will issue warnings<br/>
Benjamin Young: whole point of this thing is to catch things that might show up in activity streams. If you do consume as json-ld, you do preserve the author’s intent …<br/>
Benjamin Young: <a href="https://github.com/w3c/activitystreams-testing/issues/4">https://github.com/w3c/activitystreams-testing/issues/4</a><br/>
Gregg Kellogg: if we look at the import activity streams an set <code>@vocab</code> to some urn, that way it doesn’t lose anything and it round-trips, which maps to issue above…<br/>
Proposed resolution: closed #183 as wontfix (Benjamin Young)<br/>
Benjamin Young: +1<br/>
Benjamin Young: they use their own media type in 1.0 to support list of lists. Should there be a follow up issue to say what should be done instead of blank node?<br/>
Ivan Herman: +1<br/>
Ruben Taelman: +1<br/>
Harold Solbrig: +1<br/>
Tim Cole: +1<br/>
Resolution #2: closed #183 as wontfix<br/>
</details>

**Comment by BigBlueHat** [comment]
Turns out we shouldn't have closed this one...as it relates heavily to #37.

We can continue the conversion in #37, but wanted to tie these two issues together for posterity (at the very least).
